app/javascript/react/screens/App/Overview/Overview.js

```diff
@@ -118,7 +118,7 @@ class Overview extends React.Component {
       freshTransformationPlans.forEach(plan => {
         const mostRecentRequest = plan.miq_requests.reduce(
           (prev, current) =>
```

Comment:
since we are starting to repeat this logic a lot of places, it seems like this is a good opportunity to make it a function. Do you mind creating a simple helper in `screens/App/common/helper.js` which accepts an array of requests and returns most recent?
```
export const mostRecentRequest = (requests) => {
return requests.reduce(
    (prev, current) => (prev.updated_on > current.updated_on ? prev : current)	+    (prev, current) => (prev.created_on > current.created_on ? prev : current)
   );	   );
}
```

Reply:
otherwise..this PR looks good to me @AparnaKarve !

Reply:
Sure! That's a good idea @priley86

Reply:
@priley86 I have extracted the `mostRecentRequest` logic in the `app/javascript/react/screens/App/common/getMostRecentRequest.js` helper.

(It should be easier to review this change visually if you look at 3bde4ef)